Fix instructions for using serverless from the CLI without GitHub (#12161)
dagster-io/dagster: docs/content/dagster-cloud/deployment/serverless.mdx

You can also configure the `dagster-cloud` tool noninteractively; see [the CLI docs](/dagster-cloud/developing-testing/dagster-cloud-cli#environment-variables-and-cli-options) for more information.

> Add 'dagster-cloud' as a dependency to `my-dagster-project/setup.py`:

Reviewer: ```suggestion
Add `dagster-cloud` as a dependency to `my-dagster-project/setup.py`:
```

🔧 formatting; package/library names should be formatted as code

> ```

Reviewer: ```suggestion
```python
```

:wrench: code blocks should have a language associated with them so syntax highlighting is applied

  install_requires=[
      "dagster",
      "dagster-cloud",    # add this line

>       "dagster-airbyte",

Reviewer: I'm not sure what happened with the GH suggestion I made, but this is still indented a bit too much - could you please fix this before merging? :)

Author: my mistake probably, fixed all the way to the left now.

>     install_requires=[
>         "dagster",
>         "dagster-cloud",    # add this line
>         "dagster-airbyte",

Reviewer: ```suggestion
install_requires=[
    "dagster",
    "dagster-cloud",    # add this line
    "dagster-airbyte",
```

🔧 In this case, the code should be flush-left. You might need to run `make mdx-format` locally after making this change, or BK could throw a fit.
